js/calc.js

```diff
@@ -0,0 +1,64 @@
+function sub(a, b) {
+	return a - b;
+}
+
+function add(a, b) {
+	return a + b;
+}
+
+function mul(a, b) {
+	return a * b;
+}
+
+function div(a, b) {
+	return a / b;
+}
+
+function intdiv(a, b) {
```

Comment:
hm, it's correct, but it's incorrect in the same way
when I try to call intdiv with params 20.65, 10, I expect 2 and actual result is 2
when arguments are 20.65, 'sd', I expect NaN, but result is 0